src/arrayMethodJoin.js

```diff
@@ -4,8 +4,20 @@
  * Implement method join
  */
 function applyCustomJoin() {
-  [].__proto__.join2 = function(separator) {
-    // write code here
+  [].__proto__.join2 = function(separator = ',') {
+    let stringOutput = '';
+
+    for (let i = 0; i < this.length; i++) {
+      if ((this[i] !== undefined && this[i] !== null)) {
+        stringOutput += `${this[i]}`;
```

Comment:
won't this work without a template string here? It's just a single variable without any additional text 🤔